config.py

```diff
@@ -1,10 +1,11 @@
-import os
 # Este archivo configura la app
+import os # os interactua con el sistema operativo
+
 # Configura la cadena de conexión desde el panel de Vercel
 class Config:
-    #SQLALCHEMY_DATABASE_URI = "postgres://default:ARgfMHVes85Z@ep-white-morning-a40680zk.us-east-1.aws.neon.tech:5432/verceldb?sslmode=require"
+
     SQLALCHEMY_DATABASE_URI = (
-        "postgresql://default:ARgfMHVes85Z@ep-white-morning-a40680zk.us-east-1.aws.neon.tech:5432/verceldb?sslmode=require"
+          "postgresql://default:ARgfMHVes85Z@ep-white-morning-a40680zk.us-east-1.aws.neon.tech:5432/verceldb?sslmode=require"
```

Comment:
Quizás podrías revisar los variables de entorno, para que no subas tus accesos a la base de datos. [variables de entorno ](https://dev.to/asjordi/como-utilizar-variables-de-entorno-en-python-4pk0)